### PR TITLE
Use VARCHAR and VARBINARY types for JDBC driver

### PR DESCRIPTION
**Pull request by electrum**
Presto "varchar" is now VARCHAR rather than LONGNVARCHAR
and "varbinary" is now VARBINARY rather than LONGVARBINARY.

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java [hunk 1]
```diff
@@ -229,20 +223,20 @@ public void testSelectAllDatatypesAvro()
...
+                        "346," +
+                        "345.67800," +
+                        "'" + parseTimestampInLocalTime("2015-05-10 12:15:35.123", PRODUCT_TESTS_TIME_ZONE).toString() + "'," +
+                        "'" + Date.valueOf("2015-05-10") + "'," +
+                        "'ala ma kota'," +
+                        "'ala ma kot'," +
+                        "'ala ma    '," +
+                        "true," +
+                        "'kot binarny'" +
+                        ")",
```
Comment (kokosing) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java]: undo or extract to separate commit

Comment (electrum) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java]: Thanks for catching that. Fixed
